src/main/java/mate/academy/service/impl/MovieSessionServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.service.impl;
+
+import jakarta.persistence.EntityNotFoundException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+import mate.academy.dao.MovieSessionDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.MovieSession;
+
+@Service
+public class MovieSessionServiceImpl implements mate.academy.service.MovieSessionService {
+    @Inject
+    private MovieSessionDao movieSessionDao;
+
+    @Override
+    public MovieSession add(MovieSession movieSession) {
+        return movieSessionDao.add(movieSession);
+    }
+
+    @Override
+    public MovieSession get(Long id) {
+        return movieSessionDao.get(id).orElseThrow(() -> new EntityNotFoundException(String.format(
+                "Can't find movie session with id: %d", id)));
```

Comment:
According to the checklist, the `get()` method on the DAO layer should return `Optional`, and the entity should be returned on the service layer. Here, you are correctly returning the entity on the service layer, but you should also check that the DAO layer `get()` method is returning an `Optional` as per the task's requirements.